mmv1/products/securitycenterv2/OrganizationSource.yaml

```
@@ -0,0 +1,95 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'organization_Source'
+base_url: organizations/{{organization}}/sources
+self_link: '{{name}}'
+update_verb: :PATCH
+update_mask: true
+description: |
+  A Cloud Security Command Center's (Cloud SCC) finding source. A finding
+  source is an entity or a mechanism that can produce a finding. A source is
+  like a container of findings that come from the same scanner, logger,
+  monitor, etc.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs'
+  api: ‘https://cloud.google.com/security-command-center/docs/reference/rest/v2/organizations.sources’
+organization_iam_policy: !ruby/object:Api::Resource::IamPolicy
```

I believe this is incorrectly named
```suggestion
iam_policy: !ruby/object:Api::Resource::IamPolicy
```
Did this compile locally?

---

local setup is not yet done. We are trying to fix it.

---

Can we keep it as iam_policy.